src/averageAges.js

```diff
@@ -37,7 +44,27 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
```

Comment:
you can make reusable function `calculateAverageAge` and use it inside `calculateWomenAverageAge` and `calculateMenAverageAge`